bookworm.py

```diff
@@ -0,0 +1,92 @@
+from sqlalchemy import create_engine
+from sqlalchemy import Column, Integer, String, DateTime, Boolean, VARCHAR
+from sqlalchemy.orm import declarative_base
+from sqlalchemy.orm import sessionmaker
+from datetime import datetime
+import click
+from sqlalchemy import delete
+
+db = create_engine("sqlite:///book_worm.sqlite")
+Base = declarative_base()
+
+
+class Bookcase(Base):
+    __tablename__ = "bookcase"
+
+    id = Column(Integer, primary_key=True)
+    title = Column(String)
+    author = Column(String)
+    date = Column(DateTime)
+    # TODO jde vytvořit instance False/True, pokud ano, jak na to? - dokumentace sql, python class?
+    # unread = Column(Boolean(), nullable=False, default=True)
```

Comment:
unread je opakem pro read, to ti vlastne vyjadri kdyz u read budes mit hodnotu False ;)